python/app/db/models/order.py

```diff
@@ -8,8 +8,14 @@
 @strawberry.enum
 class OrderStatus(enum.Enum):
     PENDING = "PENDING"
+    ORDER_PLACED = "ORDER_PLACED"
+    ACCEPTED = "ACCEPTED"
     CANCELLED = "CANCELLED"
+    READY_FOR_DELIVERY = "READY_FOR_DELIVERY"
     COMPLETE = "COMPLETE"
+    SCHEDULED = "SCHEDULED"
+    PICKED_UP = "PICKED_UP"
+    DELIVERED = "DELIVERED"
```

Comment:
Whats the difference b/w COMPLETE and DELIVERED?

Reply:
both are same, need help in updating the schema to remove it. Added to todo list for now.

Reply:
okay approving this for now, buts its just a one line change right? keep either DELIVERED or COMPLETE.